fboom_me.sh

```diff
@@ -0,0 +1,201 @@
+# Plowshare fboom.me module
+# by idleloop <idleloop@yahoo.com>, v1.0, Feb 2016
+#
+# This file is part of Plowshare.
+#
+# Plowshare is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Plowshare is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Plowshare.  If not, see <http://www.gnu.org/licenses/>.
+#
+# (Official API probably similar to: https://github.com/keep2share/api)
+
+MODULE_FBOOM_ME_REGEXP_URL='https\?://\(www\.\)\?fboom.me/'
+
+MODULE_FBOOM_ME_DOWNLOAD_OPTIONS=""
+MODULE_FBOOM_ME_DOWNLOAD_RESUME=yes
```

Comment:
This service not support resuming downloads for anonymous and free accounts. They state what are the limitations. Some service not provide those information. Fortunately you can always check if they support resuming downloads. You can partially download a specific file and then trying again it download, you will see if it works or not. If you will ever implement a premium account support, then you can do what I did [here](https://github.com/mcrapet/plowshare-modules-legacy/blob/master/catshare.sh#L93). So here it should be:
```
MODULE_FBOOM_ME_DOWNLOAD_RESUME=no
```
BTW. Filejoker also don't have support for resuming downloads for anonymous and free accounts.